src/core/MixHelpers.cpp

```diff
@@ -77,7 +77,7 @@ bool sanitize( sampleFrame * src, int frames )
 	{
 		for( int c = 0; c < 2; ++c )
 		{
-			if( isinff( src[f][c] ) || isnanf( src[f][c] ) )
+			if( unlikely( isinff( src[f][c] ) || isnanf( src[f][c] ) ) )
```

Comment:
Please note that `unlikely` is incompatible with MSVC.

Reply:
Right. As I understand it this should work in master then where `Q_UNLIKELY(x)` is used instead of `builtin_expect`. I'll revert this for stable-1.2 .

Reply:
Fixed!